src/numpy.jl

```diff
@@ -170,7 +172,8 @@ const npy_typestrs = Dict( "b1"=>Bool,
                            "i8"=>Int64,     "u8"=>UInt64,
                            "f4"=>Float32,   "f8"=>Float64,
                            "c8"=>Complex64, "c16"=>Complex128,
-                           "O"=>PyPtr, "O$(div(Sys.WORD_SIZE,8))"=>PyPtr )
+                           "O"=>PyPtr, "O$(div(Sys.WORD_SIZE,8))"=>PyPtr,
+                           "f2"=>Float16)
```

Comment:
I was thinking that it will look weird as all the rest "f"s and "u"s and "i"s were paired, up but I think it looks fine now that I've moved "f2" up there.